### PR TITLE
Added various websites

### PR DESCRIPTION
**Pull request by Penaz91**
I added some websites to this updated fork after being asked if it was possible to bring the changes here, on this pull request: https://github.com/justdeleteme/justdelete.me/pull/803

Thank you for your attention.

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -360,6 +360,16 @@
             "aol.com"
         ]
     },
+    
+    {
+        "name": "Apex Minecraft Hosting",
+        "url": "https://billing.apexminecrafthosting.com/submitticket.php",
+        "difficulty": "hard",
+        "notes": "You need to log into your billing panel and send a ticket regarding the account deletion. If you only have a multicraft account, you can send a ticket without logging in."
```
Comment (tupaschoal) [inline _data/sites.json]:
```suggestion
        "notes": "You need to log into your billing panel and send a ticket regarding the account deletion. If you only have a multicraft account, you can send a ticket without logging in.",
```
Missing comma

File: _data/sites.json [hunk 2]
```diff
@@ -983,6 +993,16 @@
...
+    
+    {
+        "name": "Buycraft",
+        "url": "https://server.tebex.io/profile/delete",
+        "difficulty": "easy",
+        "notes": "Buycraft was moved to Tebex, just click the 'delete profile' button in your account menu",
+        "domains": [
+            "tebex.io"
+        ]
+    },
```
Comment (tupaschoal) [inline _data/sites.json]: Needs to come after `Buycott` to keep sorted

Comment (Penaz91) [reply _data/sites.json]: I'm sorry, seems the fever made me forget my ABCs. Should be fixed now

File: _data/sites.json [hunk 3]
```diff
@@ -7082,6 +7123,16 @@
...
+    
+    {
+        "name": "Tebex",
+        "url": "https://server.tebex.io/profile/delete",
+        "difficulty": "easy",
+        "notes": "Just click the 'delete profile' button in your account menu",
+        "domains": [
+            "tebex.io"
+        ]
+    },
```
Comment (tupaschoal) [inline _data/sites.json]: Tebex also needs to come after Teamviewer 😄

Comment (tupaschoal) [inline _data/sites.json]: Tebex also needs to come after Teamviewer 😄

File: _data/sites.json (hunk 2 above)
Comment (tupaschoal) [reply _data/sites.json]: It's alright :)

File: _data/sites.json (hunk 3 above)
Comment (Penaz91) [reply _data/sites.json]: Everything should be ok now, sorry again for this

**Comment by tupaschoal** [comment]
Don't worry, it happens. I fixed the few remaining issues.

**Pull request merged**